ci-operator/config/openshift/loki/openshift-loki-upstream-v2.9.6.yaml

```diff
@@ -6,12 +6,12 @@ base_images:
   ocp_builder_rhel-9-golang-1.21-openshift-4.16:
     name: builder
     namespace: ocp
-    tag: rhel-9-golang-1.21-openshift-4.16
+    tag: rhel-9-golang-1.21-builder-multi-openshift-4.16
```

Comment:
Please change also the name of this image here to `ocp_builder_rhel-9-golang-1.21-builder-multi-openshift-4.16` and references below